# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/modemmanager-qt/hotfix-sim.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- modemmanager-qt-6.21.0/src/sim.cpp.vanilla	2025-12-18 17:36:13.196063391 +0100
+++ modemmanager-qt-6.21.0/src/sim.cpp	2025-12-18 17:36:32.164653301 +0100
@@ -34,7 +34,7 @@
         operatorName = simIface.operatorName();
         emergencyNumbers = simIface.emergencyNumbers();
         preferredNetworks = simIface.preferredNetworks();
-#if MM_CHECK_VERSION(1, 20, 0)
+#if 0 // MM_CHECK_VERSION(1, 20, 0)
         gid1 = simIface.gid1();
         gid2 = simIface.gid2();
         simType = (MMSimType)simIface.simType();
@@ -119,7 +119,7 @@
     return d->preferredNetworks;
 }
 
-#if MM_CHECK_VERSION(1, 20, 0)
+#if 0 // MM_CHECK_VERSION(1, 20, 0)
 QByteArray ModemManager::Sim::gid1() const
 {
     Q_D(const Sim);
@@ -246,7 +246,7 @@
             preferredNetworks = it->toMap();
             Q_EMIT q->preferredNetworksChanged(preferredNetworks);
         }
-#if MM_CHECK_VERSION(1, 20, 0)
+#if 0 // MM_CHECK_VERSION(1, 20, 0)
         it = properties.constFind(QLatin1String(MM_SIM_PROPERTY_GID1));
         if (it != properties.constEnd()) {
             gid1 = it->toByteArray();
--- modemmanager-qt-6.21.0/src/sim.h.vanilla	2025-12-18 17:36:01.390440277 +0100
+++ modemmanager-qt-6.21.0/src/sim.h	2025-12-18 17:36:10.908523855 +0100
@@ -109,7 +109,7 @@
      */
     QVariantMap preferredNetworks() const;
 
-#if MM_CHECK_VERSION(1, 20, 0)
+#if 0 // MM_CHECK_VERSION(1, 20, 0)
     /*!
      * Returns Group identifier 1evel 1.
      */
